test/cases/ratings/getRating.ts

```diff
@@ -80,7 +83,7 @@ describe("getRating", () => {
                     count: 10,
                     hash: videoOneIDHash,
                 }];
-                assert.ok(partialDeepEquals(res.data, expected));
+                assert.ok(arrayPartialDeepEquals(res.data, expected));
```

Comment:
Is this due to changing order?

Reply:
yep, the order would randomly change. arrayPartialDeepEquals is a lot heavier so I only want to use it when partialDeepEquals doesn't return consistent results

Reply:
for some reason, this was the **only** test that would consistently fail for any subsequent tests on redis